src/main/java/net/neoforged/neoforge/fluids/FluidType.java

```diff
@@ -535,6 +553,28 @@ public int getViscosity(FluidState state, BlockAndTintGetter getter, BlockPos po
         return this.getViscosity();
     }
 
+    /**
+     * Returns whether a fluid above a pointed dripstone block can successfully fill a cauldron below.
```

Comment:
This javadoc isn't fully indicative of the side-effects of this method (replacing the block with the filled cauldron block, sending a game event, and playing the sound).

Also, because of those side effects, this method should probably be renamed to be more descriptive of those side effects -- `handleCauldronDrip` perhaps?